tests/unit/PostgrestFilterTest.php

```diff
@@ -27,118 +29,261 @@ public function setup(): void
     public function testEq(): void
     {
         $result = $this->filter->eq('name', 'Algeria');
+        $reflection = new ReflectionClass($result->{'url'});
+        $url = $reflection->getProperty('host');
+        $query = $reflection->getProperty('query');
+        //$params = $reflection->getProperty('parameters');
```

Comment:
This is how I'm intending on accessing the parameters array, which contains the key name,  which needs to be asserted on the unit testing